lib/Server.js

```diff
@@ -342,10 +342,11 @@ class Server {
 
   setupStaticFeature() {
     const contentBase = this.options.contentBase;
+    const contentBasePublicPath = this.options.contentBasePublicPath;
 
     if (Array.isArray(contentBase)) {
       contentBase.forEach((item) => {
-        this.app.get('*', express.static(item));
+        this.app.get(contentBasePublicPath, express.static(item));
```

Comment:
@evilebottnawi does `this.options.staticOptions` is missing here in `express.static(item)`

Reply:
I want to know, you understand what is happening in the code and what you write? I'm not trying to be toxic, I just ask because the changes look like you don’t understand what's going on here.

Reply:
No problem. `contentBase` a folder or an array of folder from where to serve content
In case of single folder `this.options.staticOptions` are passed, in case of multiple folders, are not passed.

Reply:
Ok. In docs is mentioned this
> This only works when using devServer.contentBase as a string.